apps/dolly-backend/src/main/java/no/nav/dolly/consumer/graphql/GraphQLRequest.java

```diff
@@ -0,0 +1,30 @@
+package no.nav.dolly.consumer.graphql;
+
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.apache.http.Consts;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Map;
+import java.util.stream.Collectors;
+
```

Comment:
Du kan legge til .retryWhen() for alle kallene
og kanskje feilhåndtering?
.onErrorResume()